app/journal.py

```diff
@@ -0,0 +1,53 @@
+import pymysql
+from app import app
+from db import mysql
+from flask import jsonify, request
+from flask_cors import cross_origin
+
+# Get list of journals
+@app.route("/api/journals/titles")
+@cross_origin()
+def getAvailableJournals():
+    query = """
+    SELECT * FROM Journals
+    """
+
+    conn = mysql.connect()
+    cursor = conn.cursor(pymysql.cursors.DictCursor)
+    cursor.execute(query)
+    results = cursor.fetchall()
+
+    resp = jsonify(results)
+
+    resp.status_code = 200
+
+    return resp
+
+#Get all articles of a specific journal
+@app.route("/api/journals")
+@cross_origin()
+def getSpecificArticlesfromJournals():
+    query_parameters = request.args
+    journal = query_parameters.get('journalName')
+
+    query = """
+    SELECT Title, URL, DOI, YearPublished, JournalName, CategoryName, AuthorInitial, AuthorSurname
```

Comment:
can you make this SQL query neater please, follow what I did with a more complex SQL query on system-design-backend

Reply:
I have applied the changes you requested.